docs/integrations/builtin/trigger-nodes/n8n-nodes-base.seatabletrigger.md

```diff
@@ -7,7 +7,7 @@ contentType: [integration, reference]
 
 # SeaTable Trigger node
 
-[SeaTable](https://seatable.co){:target=_blank .external-link} is a collaborative database application with a spreadsheet interface.
+[SeaTable](https://seatable.com){:target=_blank .external-link} is a collaborative database application with a spreadsheet interface.
```

Comment:
Should this maybe be https://seatable.io/en/ ?

Reply:
Nice spot, i thought it was just a typo